container.css

```diff
@@ -0,0 +1,40 @@
+.subcontainer{
+    display:var(--container-display);
+    padding: 0px 50px 40px 200px;
+    height: 100%;
+}
+.categories{
+    display: var(--container-display);
+    flex-direction: var(--container-flex-direction);  
+    padding:0px 200px 0px 100px;
+    
+}
+
+.catheading{
+    margin-bottom: 20px;
+    font-size: 30px;
+    font-weight: var(--container-font-weight);
+}
+.catheading--btn{
+    font-size: 18px;
+}
+.info{
+    display: var(--container-display);
+    width:500px;
+    height: 600px;
+    padding: 25px 50px 50px 50px;
+    border: 3px solid #2166e6;
+    border-radius: 8px;
+}
+.info__infolabl{
+display:var(--container-display);
```

Comment:
u dont need to create variables for every thing, basic thing like font size, colour, background these can have variables. the idea is create variable for only things which you think are part of theme.